oktaawscli/okta_auth.py

```diff
@@ -142,6 +154,40 @@ def verify_single_factor(self, factor, state_token):
                         exit(1)
                     else:
                         time.sleep(0.5)
+
+            if factor['factorType'] == 'u2f':
+                devices = u2f.list_devices()
+                if len(devices) == 0:
+                    print("No U2F device found")
```

Comment:
We should have any output to the console running through the logger, instead of `print()`.
```suggestion
                    self.logger.warning("No U2F device found")
```

Reply:
I think that's from copy'n'paste as the MFA_CHALLENGE is using `print` as well

Reply:
done

Reply:
Yeah, I noticed that as well. I need to fix that. Thanks!